src/test/kotlin/com/coditory/gradle/integration/acceptance/SpockBasedAcceptanceTest.kt

```diff
@@ -17,13 +17,13 @@ class SpockBasedAcceptanceTest {
             }
 
             repositories {
-                jcenter()
+                mavenCentral()
             }
 
             dependencies {
                 testImplementation "junit:junit:4.12"
-                testImplementation "org.codehaus.groovy:groovy-all:2.4.13"
-                testImplementation "org.spockframework:spock-core:1.1-groovy-2.4"
+                testImplementation "org.codehaus.groovy:groovy-all:3.0.22"
```

Comment:
maybe go to groovy4?

Reply:
and groovy-all should not be required since spock already comes with groovy

Reply:
Sounds about right

Reply:
I'll change it a separate PR